frontend/src/pages/modelRegistrySettings/CreateModal.tsx

```diff
@@ -88,7 +89,8 @@ const CreateModal: React.FC<CreateModalProps> = ({ isOpen, onClose }) => {
       },
     };
     try {
-      await createModelRegistryBackend(data);
+      await createModelRegistryBackend({ modelRegistry: data, databasePassword: password });
```

Comment:
Remind me of talking about the posibility of using descriptive names for model registries, we sholud have a display name option, but we should talk with ux first.